src/sentry/ingest/transaction_clusterer/tree.py

```diff
@@ -80,7 +84,7 @@ def __init__(self, *, merge_threshold: int) -> None:
 
     def add_input(self, strings: Iterable[str]) -> None:
         for string in strings:
-            parts = string.split(SEP)
+            parts = string.split(SEP, maxsplit=MAX_DEPTH)
```

Comment:
meaning we get only 200 segments here and the rest will be ignored?

Reply:
The remaining segments will be a leaf node in the tree, e.g. `/1/2/3/4/5/../200/201/202/203` will become the path `["1", "2", "3", "4", "5", .., "200/201/202/203"]`.